frontend/app/components/wp-table/timeline/wp-timeline.dummy-control.directive.ts

```diff
@@ -60,31 +60,37 @@ const template = `
 
 class WorkPackageTimelineControlController {
 
-  hscroll: number;
+  private wpTimeline: WorkPackageTimelineTableController;
 
+  hscroll: number;
   zoom: string;
 
-  /*@ngInject*/
-  constructor(private workPackageTimelineService: WorkPackageTimelineService) {
-    this.hscroll = workPackageTimelineService.viewParameterSettings.scrollOffsetInDays;
+  constructor() {
+  }
+
+  $onInit() {
```

Comment:
Note: Changed to `$onInit` due to the additional binding to wpTimeline which is not yet available in the constructor